<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rcl::gui::cl::CoverLetterTab</class>
 <widget class="QWidget" name="rcl::gui::cl::CoverLetterTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="leftWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QScrollArea" name="experienceScrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="rcl::gui::cl::experience::ExperiencePane" name="experienceScrollAreaContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>179</width>
           <height>438</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="writeBtn">
        <property name="text">
         <string>Write Cover Letter</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="coverLetterScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="coverLetterScrollAreaContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>256</width>
        <height>468</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="rightWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="rightInternalWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QScrollArea" name="preambleScrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="rcl::gui::cl::preamble::PreamblePane" name="preambleScrollAreaContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>179</width>
              <height>215</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="propertiesScrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="rcl::gui::cl::properties::PropertiesPane" name="propertiesScrollAreaContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>179</width>
              <height>215</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="printBtn">
        <property name="text">
         <string>Print</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>rcl::gui::cl::properties::PropertiesPane</class>
   <extends>QWidget</extends>
   <header>property/propertiespane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>rcl::gui::cl::preamble::PreamblePane</class>
   <extends>QWidget</extends>
   <header>preamble/preamblepane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>rcl::gui::cl::experience::ExperiencePane</class>
   <extends>QWidget</extends>
   <header>experience/experiencepane.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
